src/theme/theme.ts

```diff
@@ -22,6 +22,8 @@ const palette = {
   purple: '#B377FF',
   purpleLight: '#6C71A3',
   purpleMain: '#474DFF',
+  purple300: '#343964',
```

Comment:
@matthewcarlreetz I was thinking we might want to move to a color naming scheme like this. Sort of like how ant.design does it (https://ant.design/docs/spec/colors#header). I think we'll end up in a painful place if we try to classify all the different color variants by dark, darker, darkest etc. It'd also maybe be good to use storybook or even something off-app to preview the different colors. I can maybe look into that.

Reply:
I TOTALLY agree :)

Reply:
@allenan I've been meaning to ask @PeterMain about getting a color palette. 

I'm most familiar with https://material.io/design/color/the-color-system.html#color-theme-creation, but I'm up for whatever.